applications/conceptual-model-editor/src/app/diagram/visualization.tsx

```diff
@@ -522,18 +581,19 @@ export const Visualization = () => {
                     />
                     <Panel position="top-right">
                         <div className="flex flex-row-reverse">
-                            <div className="cursor-help" title="add class to canvas by clicking and holding `alt`">
+                            <div className="cursor-help" title="Add class to canvas by clicking and holding `alt`">
                                 ℹ
                             </div>
                             <div className="mx-1">
-                                <button title="download svg of the canvas" onClick={exportCanvasToSvg}>
+                                <button title="Download svg of the canvas" onClick={exportCanvasToSvg}>
```

Comment:
Try to move towards using the localization file. Our goal should be to get all strings outside of the components code.